robot_smach_states/src/robot_smach_states/manipulation/grab.py

```diff
@@ -206,7 +206,7 @@ def execute(self, userdata=None) -> str:
         if not arm.send_goal(goal_bl, timeout=0.0, allowed_touch_objects=[grab_entity.uuid]):
             rospy.logerr('Failed retract')
         arm.wait_for_motion_done()
-        self.robot.base.force_drive(-0.125, 0, 0, 2.0)
+        self.robot.base.force_drive(-0.2, 0, 0, 2.0)
```

Comment:
This is a bigger change in behaviour. I would reverse this and we can tune it again if it becomes a problem.